In [2]:
import sys
sys.path.append("D:/projects/zsc")



In [9]:
sys.path.append("D:/projects/zsc/T2ICount/")
sys.path.append("D:/projects/zsc/T2ICount/taming-transformers/")
!python D:/projects/zsc/T2ICount/visualize.py

Loading model from configs/v1-5-pruned-emaonly.ckpt
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
making attention of type 'vanilla' with 512 in_channels
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
making attention of type 'vanilla' with 512 in_channels
--- debug features ---
img_feat[0] shape: torch.Size([2, 320, 48, 48])
img_feat[1] shape: torch.Size([2, 640, 24, 24])
img_feat[2] shape: torch.Size([2, 1280, 12, 12])
img_feat[3] shape: torch.Size([2, 1280, 6, 6])
attn12 shape: torch.Size([2, 77, 12, 12])
attn24 shape: torch.Size([2, 77, 24, 24])
attn48 shape: torch.Size([2, 77, 48, 48])
text_feat shape: torch.Size([2, 77, 768])
Predicted Number: 8.732687


d:\conda_envs\zsc\lib\site-packages\huggingface_hub\file_download.py:949: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(


Loading model from configs/v1-5-pruned-emaonly.ckpt
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
making attention of type 'vanilla' with 512 in_channels
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
making attention of type 'vanilla' with 512 in_channels
--- debug features ---
img_feat[0] shape: torch.Size([2, 320, 48, 48])
img_feat[1] shape: torch.Size([2, 640, 24, 24])
img_feat[2] shape: torch.Size([2, 1280, 12, 12])
img_feat[3] shape: torch.Size([2, 1280, 6, 6])
attn12 shape: torch.Size([2, 77, 12, 12])
attn24 shape: torch.Size([2, 77, 24, 24])
attn48 shape: torch.Size([2, 77, 48, 48])
text_feat shape: torch.Size([2, 77, 768])
Predicted Number: 13.320862
d:\conda_envs\zsc\lib\site-packages\huggingface_hub\file_download.py:949: FutureWarning: `resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.
  warnings.warn(